src/hard_sphere/mod.rs

```diff
@@ -3,10 +3,9 @@
 use feos_core::{HelmholtzEnergyDual, StateHD};
 use ndarray::*;
 use num_dual::DualNum;
-use std::borrow::Cow;
+use std::{borrow::Cow, sync::Arc};
 use std::f64::consts::FRAC_PI_6;
 use std::fmt;
-use std::rc::Rc;
 
```

Comment:
```suggestion
use std::sync::Arc
```
wanted to try the suggestion feature :)